packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -119,14 +156,84 @@ export function serveWebpackBrowser(
       browserName,
     );
 
-    const webpackConfigResult = await buildBrowserWebpackConfigFromContext(
+    const { config, projectRoot, i18n } = await buildBrowserWebpackConfigFromContext(
       browserOptions,
       context,
       host,
+      true,
     );
+    let webpackConfig = config;
+
+    const tsConfig = readTsconfig(browserOptions.tsConfig, context.workspaceRoot);
+    if (i18n.shouldInline && tsConfig.options.enableIvy !== false) {
+      if (i18n.inlineLocales.size > 1) {
+        throw new Error(
+          'The development server only supports localizing a single locale per build',
+        );
+      }
+
+      const locale = [...i18n.inlineLocales][0];
+      const translation = i18n.locales[locale] && i18n.locales[locale].translation;
+
+      const { plugins, diagnostics } = await createI18nPlugins(
+        locale,
+        translation,
+        browserOptions.i18nMissingTranslation,
+      );
 
-    // No differential loading for dev-server, hence there is just one config
-    let webpackConfig = webpackConfigResult.config;
+      // Get the insertion point for the i18n babel loader rule
+      // This is currently dependent on the rule order/construction in common.ts
+      // A future refactor of the webpack configuration definition will improve this situation
+      // tslint:disable-next-line: no-non-null-assertion
+      const rules = webpackConfig.module!.rules;
+      const index = rules.findIndex(r => r.enforce === 'pre');
+      if (index === -1) {
+        throw new Error('Invalid internal webpack configuration');
+      }
+
+      const i18nRule: webpack.Rule = {
+        test: /\.(?:m?js|ts)$/,
+        exclude: /[\/\\](?:core-js|@babel|tslib)[\/\\]/,
```

Comment:
Are these exclusions just a performance thing or does something break in those?

Also, I find the babel exclusion odd. I didn't expect any babel code to be bundled in the application.

Reply:
Performance.  There is no possibility of those containing `$localize` so no point in parsing and processing each one.  The loader rule was moved over from my usage-based polyfills branch and the regex along with it.

Reply:
Removed the exclude.  Not really needed so no real reason to add to the complexity.